src/main/java/frc/robot/Ids.java

```diff
@@ -4,10 +4,10 @@
     public static final int CONTROLLER_DRIVE_PORT = 0;
     public static final int CONTROLLER_OPERATOR_PORT = 0;
 
-    public static final int DRIVE_LEFT_LEAD_DEVICE = 1;
-    public static final int DRIVE_LEFT_FOLLOW_DEVICE = 2;
-    public static final int DRIVE_RIGHT_LEAD_DEVICE = 4;
-    public static final int DRIVE_RIGHT_FOLLOW_DEVICE = 3;
+    public static final int DRIVE_LEFT_LEAD_DEVICE = 3;
+    public static final int DRIVE_LEFT_FOLLOW_DEVICE = 4;

```

Comment:
Just noticed that switching these around might mean we don't have to do `rightMotors.setInverted()`. Let's test that before merging.